code/native/mpris.py

```diff
@@ -344,16 +354,17 @@ class StreamKeysMPRIS(MediaPlayer):
 
     def __setattr__(self, name, value):
```

Comment:
I think that this `__setattr__()` method here is not needed anymore. Perhaps remove it?